From 1de949e6248271ebab7cb1c77ab2ae249bbe8c7e Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Mon, 15 Aug 2022 19:42:38 +0100
Subject: [PATCH] drm/panel-simple: hack ignore orientation

downstream removal of orientation plus upstream adding of new function
---
 drivers/gpu/drm/panel/panel-simple.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/panel/panel-simple.c b/drivers/gpu/drm/panel/panel-simple.c
index 1705b34ef76b..2e21f5075edb 100644
--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -434,9 +434,9 @@ static int panel_simple_get_timings(struct drm_panel *panel,
 
 static enum drm_panel_orientation panel_simple_get_orientation(struct drm_panel *panel)
 {
-	struct panel_simple *p = to_panel_simple(panel);
+	//struct panel_simple *p = to_panel_simple(panel);
 
-	return p->orientation;
+	return DRM_MODE_PANEL_ORIENTATION_UNKNOWN;
 }
 
 static const struct drm_panel_funcs panel_simple_funcs = {
-- 
2.35.5

